core/sql/comexe/ComTdbExeUtil.h

```diff
@@ -2845,35 +2846,36 @@ class ComTdbExeUtilLobShowddl : public ComTdbExeUtil
 
   short getLOBnum(short i);
   char * getLOBloc(short i);
-
+  NABoolean getIsExternalLobCol(short i);
   UInt16 numLOBs() { return numLOBs_; }
 
   char * getLOBnumArray() { return lobNumArray_; }
   char * getLOBlocArray() { return lobLocArray_; }
-
+  char * getLOBtypeArray() { return lobTypeArray_;}
+ 
   char * schName() { return schName_; };
 private:
-  UInt32 flags_;                                     // 00-03
+  UInt32 flags_;                                    
 
-  UInt16 numLOBs_;                                   // 04-05
+  UInt16 numLOBs_;                                  
   short maxLocLen_;
   
   // array of shorts. numLOBs entries. 
   // Each entry is the lobNum.
-  NABasicPtr lobNumArray_;                           // 08-15
+  NABasicPtr lobNumArray_;                           
 
   // array of string, null terminated. numLOBs entries. 
   // Each entry is the storage location of lob data file.
-  NABasicPtr lobLocArray_;                           // 16-23
-
-  Int64 objectUID_;                                  // 24-31
+  NABasicPtr lobLocArray_;                          
+  //array of int32. numLOBs entries
+  NABasicPtr lobTypeArray_; 
+  Int64 objectUID_;                                  
 
-  NABasicPtr schName_;                               // 32-39
-  short schNameLen_;                                 // 40-41
+  NABasicPtr schName_;                               
+  short schNameLen_;                                
 
-  short sdOptions_;                                  // 42-43
-
-  char fillersComTdbExeUtilLobShowddl_[4];           // 44-47
```

Comment:
I guess we don't have to count bytes in TDBs anymore since all SQL is dynamic now. :-)